composer.json

```diff
@@ -37,7 +37,8 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Faker\\Test\\": "test/Faker/"
+            "Faker\\Test\\": "test/Faker/",
+            "Faker\\Test\\Fixture\\": "test/Fixture/"
```

Comment:
I've introduced a new namespace mapping here - we might want to use PSR-4 mapping with a flattened directory structure for the tests at some point as well!